### ISSUE TITLE
IDE0055 doesn't work well with IDE0190/csharp_style_prefer_parameter_null_checking

### ISSUE THREAD

**Issue by Evangelink** [dotnet/roslyn#59414]
**Version Used**:  VS17 stable or preview.

**Steps to Reproduce**:

```csharp
public class MyClass
{
    public MyClass(FileStream stream!!) // IDE0055 incorrect formatting on `stream!!`
    {
    }
}
```

**Expected Behavior**: No `IDE0055` when using this new feature.

**Actual Behavior**: `IDE0055` is wrongly reported.

**Comment by Youssef1313** [comment]
@RikkiGibson Was this already fixed?

**Comment by Youssef1313** [comment]
Yup found the issue: https://github.com/dotnet/roslyn/issues/58798

**Comment by jinujoseph** [comment]
@RikkiGibson  to confirm and close

**Comment by RikkiGibson** [comment]
I think it's the same issue. You have to make sure to use a new version of the formatting analyzer when you're using !!. Thanks @Youssef1313 for finding it.
